src/05-invoicing/invoices.apib

```diff
@@ -83,6 +83,7 @@ Get a list of invoices.
                     + due (Money)
                 + created_at: `2016-02-04T16:44:33+00:00` (string)
                 + updated_at: `2016-02-05T16:44:33+00:00` (string)
+                + `web_url`: `https://app.teamleader.eu/invoice_detail.php?id=4b0f959a-27e3-4080-b1b2-a05b374863cb` (string)
```

Comment:
B: let's use the actual UUID of the invoice (as returned in the `id` property) instead of a contact UUID.
```suggestion
                + `web_url`: `https://app.teamleader.eu/invoice_detail.php?id=2aa4a6a9-9ce8-4851-a9b3-26aea2ea14c4` (string)
```